docs/conf/australia/2020/news/tickets-closing-welcome-wagon.rst

```diff
@@ -0,0 +1,57 @@
+:template: {{year}}/generic.html
+
+.. post:: November 24, 2020
+   :tags: {{year}}, australia-2020, tickets
+
+Ticket Sales Closing Soon and Welcome Wagon Guide
+==================================================
+
+G'day everyone!
+
+We're just over a week away from the start of the conference,
+and we're getting excited to invite you into our online venue for 2 days of content, collaboration, and conversation.
+We will be closing ticket sales this **Friday, 27th November**, so if you've been on the fence, now is the time to buy one!
+
+If you were hoping to come to this years event,
+we recommend that you `get your tickets soon <https://www.writethedocs.org/conf/australia/2020/tickets/>`_.
+
+We have done our internal testing of the platform, and we're getting everything ready for prime time.
+As part of our internal testing, we've built out the Welcome Wagon Guide.
+This guide is our attempt to make sure that you get the most out of our events, and we've totally rewritten it for our online event.
+
+Welcome Wagon Guide
+-------------------
+
+Each year we create a guide to help our attendees get the most out of the conference.
+This year, we're all first-timers at an online event, so we've totally rewritten `our Welcome Wagon guide <https://www.writethedocs.org/conf/australia/2020/welcome-wagon/>`_ to help you get the most out of our event this year.
+
+We have a lot of information about how to use Hopin, and lots of tips and tricks for the best way to plan your days.
+
+We've taken all the transit and travel information out, but please do plan to spend a large part of your day with us from the comfort of your own home.
+
+Access to online platform
+--------------------------
+
+Once we close out our tickets, we plan to send everyone a link to access the Hopin event.
+We plan to send this out by this weekend, to give everyone a few days to work through any troubleshooting issues that you might encounter.
+The Hopin access links will be sent to all Ti.to ticketholders, so monitor your inbox later this week for a message.
+
+If you don't get an email from us by the early next week, please email us and we'll help you out!
+Once you've got your Hopin ticket, you will be ready to get started on Thursday morning!
+
+Thanks so much to all our sponsors
+----------------------------------
+
+We are so grateful to have our sponsors help in bringing these events to life every year.
+
+Thanks so much to the following companies for supporting the Australia and India conference this year:
+
+.. datatemplate::
+   :source: /_data/{{shortcode}}-{{year}}-config.yaml
+   :template: {{year}}/sponsors-simplelist.rst
```

Comment:
We have Red Hat to add here, I'll do that in another PR, but we should do it prior to shipping.

Reply:
RH PR is merged 👍